tests/epyccel/test_epyccel_mod.py

```diff
@@ -12,8 +12,8 @@
     RTOL = 1e-13
     ATOL = 1e-14
 elif os.environ.get('PYCCEL_DEFAULT_COMPILER', 'GNU') == 'intel':
-    RTOL = 1e-11
-    ATOL = 1e-14
+    RTOL = 1e-10
+    ATOL = 1e-10
```

Comment:
These tolerances are huge compared to the default ones for GNU. Are we using excessive optimizations with the Intel compilers?

Reply:
Maybe. I'm not sure. I suspect they have a different algorithm as it is only the modulo operator that gives such different values